lib/components/map/default-map.js

```diff
@@ -59,41 +119,65 @@ class DefaultMap extends Component {
       // Determine any added/removed modes
       const oldModes = oldQuery.mode.split(',')
       const newModes = newQuery.mode.split(',')
-      const removed = oldModes.filter(m => !newModes.includes(m))
-      const added = newModes.filter(m => !oldModes.includes(m))
-      const overlayVisibility = {}
+      const removed = oldModes.filter((m) => !newModes.includes(m))
```

Comment:
Should removing modes via the `ModeButtons` in the mode selector here: 
<img width="434" alt="image" src="https://user-images.githubusercontent.com/63798641/168172379-a18a6eeb-470a-47d6-9f59-7d357af34366.png">
influence the added/removed modes? I don't seem to be able to change the values of `added` or `removed` and can't untoggle the park n ride layer automatically by removing the car mode for example. 
![Route duplication bug](https://user-images.githubusercontent.com/63798641/168172852-ad1f8fb2-54a4-4977-939b-37810e2e8541.gif)

Reply:
This behavior is for the tabbed interface when you switch to/from the "Transit + Bikeshare" or "Transit + E-Scooter" travel mode. I don't think this behavior exists for the batch interface yet.

Reply:
For the batch interface, the behavior does not really apply because the query parameters (almost) never include the rental modes. However, one thing we could do in the future to implement the behavior is to show the appropriate rental overlay if the corresponding mode is the only mode active in the mode selector.

Reply:
I added a comment clarifying the behavior in https://github.com/opentripplanner/otp-react-redux/pull/606/commits/ba41be35d9aced13d5375c2544e525644918d480.